### ISSUE TITLE
Error: Comandsender link severed how to fix?

### ISSUE THREAD

**Issue by TheStarFalls** [MassiveCraft/Factions#739]
When i create a faction then try to claim land it says,
Error: Your "commandsender link" has been severed this is not supposed to happen, then it tells you relog and it could fix it but it doesn't we've tried everything to figure out what it is but we can't figure it out, What is wrong and how can i fix it so i can claim land? 
![2014-12-20_13 57 58](https://cloud.githubusercontent.com/assets/8620542/5517089/7c6c268e-8899-11e4-8a3e-8573be129678.png)

**Comment by harati** [comment]
Awesome fix : D

**Comment by JuiceHead** [comment]
Was wondering if there is in fact a fix for this yet. I have been encountering this same problem, in the same circumstances for the past few weeks.

**Comment by oloflarsson** [comment]
Cauldron is very troublesome to support. @magnusulf could you try and fix this bug?

**Comment by makkarpov** [comment]
With your 6 megabytes of junk code and excess layers of abstractions everything is very troublesome to support.

**Comment by JuiceHead** [comment]
So I guess I am just going to have to replace factions on my server. I am still getting this problem and it seems this ticket is large in part dead. Well it was nice while it lasted.

**Comment by oloflarsson** [comment]
This ticket is on our TODO list. It's not at the very top though. Factions is open source. Feel free to fix Cauldron support yourselves if you are in a hurry. I would really love such a pull request!

**Comment by ciroreed** [comment]
For those who have problems with this issue i recommend downgrade the version of factions. It seems to work fine on my server. I used for 2 months

**Comment by makkarpov** [comment]
Deleted factions from my server, this bug seems to be resolved now.

**Comment by lithiumtoxin** [comment]
It's been 3 months is there a solution to this issue yet? When I delete mstore, specifically /mstore/factions_mplayer it fixes it on first launch, but then upon restarting the server it continues to give the same error. I've tried downgrading to Factions 2.7.4 as well as 2.3.0 and a few others but then the "/f" command itself stops being recognized despite f being set as the alias in the configs and /pl showing factions as one of the plugins.

**Comment by oloflarsson** [comment]
The plugin is open source. I'd really appreciate a pull request from someone. I personally don't use cauldron. Cauldron also seems a bit dead: http://cauldron.minecraftforge.net/. They have removed their downloads. Is Cauldron even available for 1.7?

**Comment by makkarpov** [comment]
Bukkit and all it's forks seems a bit dead due to DMCA takedown.

**Comment by lithiumtoxin** [comment]
Yeah, from my understanding bukkit contained original minecraft code and so everything built off of it got taken down as well. They're working on Sponge (/www.spongepowered.org) but it's still in its infancy. When able to ill edit the specific versions im using for everything in my post above.

**Comment by lithiumtoxin** [comment]
Or I apparently can't edit posts on my phone...

I'm running 
FTB Infinity 1.2.1
Cauldron: version 1.1236.05.211
Factions 2.7.3
Massivecore 2.7.3

And these specific versions seem to be the only ones that work for me.

**Comment by Ionaru** [comment]
On Factions 2.7.5 this issue still exists. (Cauldron-MCPC-Plus version git-Cauldron-Reloaded-1.7.10-1.1291.01.0). It only happens to me when I do not have Vault installed. With Vault it works just fine as far as I can tell.

**Comment by draksterau** [comment]
Hi Guys,

This is why by the looks: Plugin Factions v2.7.5 has failed to register events for class com.massivecraft.factions.spigot.EngineSpigot because org/bukkit/event/player/PlayerInteractAtEntityEvent does not exist.


Happens in v1.7.10 Cauldron...

**Comment by draksterau** [comment]
@ulumulu1510 I think that you guys depend on v1.8 in 2.7.5? See above.

**Comment by ulumulu1510** [comment]
@draksterau
This Event is just present in spigot builds, hence it is in the spigot engine. Our engines and it's listeners get registered automatically. In the case of the SpigotEngine, this only happens if we are actually using a spigot build and are able to find this event. If it does not find it , the engine is never run.

But to your question: We compiled against 1.8, yes.

**Comment by draksterau** [comment]
Vote 1 for you to fix hero chat in 2.7.3 and release its a final 1.7.10 build?

**Comment by ulumulu1510** [comment]
@Bysokar, we will not apply every patch/commit we made since 2.7.3 to it and release a build for that. Not only is it really time inefficient, but it would also let people expect we actually do that for all the version, which we will not.

I have not the lead in Factions, but "I" wont do it. Factions is opensource though! You can fork the project, reset the git commits till 2.7.3 and apply this one patch yourself. But that is up to you, really.

**Comment by Bysokar** [comment]
It's fine. 2.5 works. I'll just hope you fix the issue  in 2.75 or higher for cauldron users.

**Comment by magnusulf** [comment]
@Bysokar You can get the error
com.massivecraft.factions.spigot.EngineSpigot because org/bukkit/event/player/PlayerInteractAtEntityEvent does not exist.
and eveything will work fine. The error was expected but not supposed to show up. So everyhting will work fine on 1.7.10.
Cauldron however is another story due to the particle effect library and stuff.

**Comment by oloflarsson** [comment]
Minecraft version is now at 1.8.7 and Cauldron is still not updated. It seems Cauldron is dead software at this point. We will not support it, sorry.

**Comment by makkarpov** [comment]
Whole bukkit looks like on big dead software.

**Comment by Bysokar** [comment]
Shame. A lot of people use 1.7.10 still ! :)

**Comment by oloflarsson** [comment]
Bukkit is not dead. Spigot updates frequently and works extremely well :smile_cat:. Long live Spigot!

**Comment by jimmboy48** [comment]
I hope someone here can maybe help. I have read from this page that faction 2.3 works with cauldron but I can't get it to work at all. I have tired 2.3 and I have change the massivecore/mcore to the correct version for the faction plugin. When I get it to loud on some of the faction/massivecore versions, everything seems to work, until i try to claim land. It tells me every time "You Can't Claim More Land, You Need More Power." I gave myself 100.0 power for me and the faction and it still says the same thing. I don't know if I am doing something wrong, and if I am please tell me what. Thanks for your time, -Bob

**Comment by Bysokar** [comment]
You cannot give yourself more power with commands in 2.3.
You have to wait. You can only increase your maximum power.

Perhaps during testing you set the interval config value for power increases to per minute ?
